eng/Versions.props

```diff
@@ -103,6 +103,20 @@
     <!-- Dependencies from https://github.com/NuGet/NuGet.Client -->
     <NuGetVersioningPackageVersion>5.8.0</NuGetVersioningPackageVersion>
   </PropertyGroup>
+  <PropertyGroup>
+    <!-- Automated versions for asp.net templates -->
+    <!-- Grab just the patch version from MicrosoftNETSdkPackageVersion (7.0.103-servicing becomes 03) -->
+    <MicrosoftNETSdkPatchVersion>$(MicrosoftNETSdkPackageVersion.Split('.')[2])</MicrosoftNETSdkPatchVersion>
+    <MicrosoftNETSdkPatchVersion>$(MicrosoftNETSdkPatchVersion.Split('-')[0])</MicrosoftNETSdkPatchVersion>
+    <MicrosoftNETSdkPatchVersion>$(MicrosoftNETSdkPatchVersion.Substring(1))</MicrosoftNETSdkPatchVersion>
+    <!-- 
+      Between branding and shipping, the templates should stay at last month's version.
+      If the incoming SDK version is 2 versions behind us, we know we just branded but haven't done the internal -> public merge yet.
+      Therefore we stay at last month's version.
+    -->
+    <AspNetCoreTemplateFeature60>$([MSBuild]::Add($(VersionFeature), 9))</AspNetCoreTemplateFeature60>
+    <AspNetCoreTemplateFeature60 Condition="$([MSBuild]::Subtract($(VersionFeature), $(MicrosoftNETSdkPatchVersion))) &lt; 2">$([MSBuild]::Add($(AspNetCoreTemplateFeature60), 1))</AspNetCoreTemplateFeature60>
```

Comment:
Note we can't just do `VersionFeature` - `MicrosoftNETSdkPatchVersion` + `N` because `VersionFeature` - `MicrosoftNETSdkPatchVersion` could be `2` (public branch, between branding and patch tuesday), `1` (public branch after patch tuesday), or `0` (internal branch after dependency flow starts), and we want `AspNetCoreTemplateFeature60` to be the same in the `1` and `0` cases